dask_cuda/tests/test_proxify_host_file.py

```diff
@@ -129,3 +129,59 @@ def test_cudf_get_device_memory_objects():
     ]
     res = get_device_memory_objects(objects)
     assert len(res) == 4, "We expect four buffer objects"
+
+
+def test_externals():
```

Comment:
The loose use of `itemsize` in the tests here seems error-prone. Unless of course I'm misunderstanding the real meaning of `itemsize`, wouldn't it be safer to do something analogous to the snippet below?

```python
dtype = np.float64
itemsize = dtype(1).itemsize
dhf = ProxifyHostFile(device_memory_limit=itemsize)
dhf["k1"] = cupy.arange(1, dtype=dtype) + 1
```

Reply:
To make it more clear, I am now defining:
```python 
cupy = pytest.importorskip("cupy")
cupy.cuda.set_allocator(None)
one_item_array = lambda: cupy.arange(1)
one_item_nbytes = one_item_array().nbytes
```